sessions/DBT-100/DBT_Workshop/jaffle_shop/README.md

```diff
@@ -485,7 +499,7 @@ having sum(number_of_orders) > 0
 
 #### Exercise 5
 
-To configure source freshness you need to update `models/sources.yml` with the following code
+To configure source freshness you need to update `models/staging/sources.yml` with the following code
```

Comment:
For my understanding, is there a reason for this not to be `models/sources.yml` instead?
I think this is simpler to start with